docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+  godbo:
```

Comment:
It's good to add `container_name` for more easily recognizing this container.
We can also add `hostname` to make it more cooler. :D

Reply:
Added. 🤩